From: Kefeng Wang <wangkefeng.wang@huawei.com>
Date: Tue, 27 Sep 2016 11:57:18 +0800
Subject: arm64: Kconfig: remove SMP dependence for NUMA
Git-commit: b4b9551e2f467d1a96c324b8a45f9b5b87d2f0fc
Patch-mainline: v4.9-rc1
References: fate#322061

The arm64 forces CONFIG_SMP=y with commit 4b3dc9679cf7, no need to
add SMP dependence for NUMA.

Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Will Deacon <will.deacon@arm.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 4572f00..17c14a1d 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -579,7 +579,6 @@ config HOTPLUG_CPU
 # Common NUMA Features
 config NUMA
 	bool "Numa Memory Allocation and Scheduler Support"
-	depends on SMP
 	select ACPI_NUMA if ACPI
 	select OF_NUMA
 	help
-- 
2.6.6

